lib/plugins/aws/provider/awsProvider.test.js

```diff
@@ -51,6 +53,73 @@ describe('AwsProvider', () => {
       delete process.env.proxy;
     });
 
+    it('should set AWS ca single', () => {
+      process.env.ca = '-----BEGIN CERTIFICATE-----\n-----END CERTIFICATE-----';
+      const newAwsProvider = new AwsProvider(serverless, options);
+
+      expect(typeof newAwsProvider.sdk.config.httpOptions.agent).to.not.equal('undefined');
+
+      // clear env
+      delete process.env.ca;
```

Comment:
this should be done in an `afterEach` block.

Reply:
Would you like this done even though it's not applicable for all tests within this describe() block?